modules/virt-creating-data-volumes-using-storage-api.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-storage-api_{context}"]
+
+= Creating data volumes using the Storage API
+
+When you create a data volume using the *storage* API, the CDI optimizes your PVC allocation based on the type of storage supported by your selected storage class. You only have to specify the data volume name, namespace, and the amount of storage that you want to allocate.
+
+For example:
+
+* When using Ceph RBD, `accessModes` is automatically set to `ReadWriteMany`, which enables Live Migration. `volumeMode` is set to `Block` to maximize performance.
+* When you are using `volumeMode` `filesystem`, more space will automatically be requested by the CDI, if required to accommodate filesystem overhead.
+
+In the following YAML, using the *storage* API requests a data volume with two gigabytes of usable space. The user does not need to know the `volumeMode` in order to correctly estimate the required PVC size. The CDI chooses the optima combination of `accessModes` and `volumeMode` attributes automatically.  These optimal values are based on the type of storage or the defaults that you define in your storage profile. If you want to provide custom values, they override the system-calculated values.
+
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: <datavolume> <1>
+spec:
+  source:
+    pvc: <2>
+      namespace: "<source-namespace>" <3>
+      name: "<my-vm-disk>" <4>
+  storage: <5>
+    resources:
+      requests:
+        storage: <2Gi> <6>
```

Comment:
Can this replaceable be something more generic instead of a specific value? Same for the `<2Gi>` in the other table.

Reply:
removed <>s and changed call-out to read "Specifies the amount..."